models/Pokedex.go

```diff
@@ -0,0 +1,33 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/dannegm/academy-go-q12021/constants"
+	"github.com/dannegm/academy-go-q12021/helpers"
+)
+
+// Pokedex stores a map of Pokemon by ID
+type Pokedex map[int]Pokemon
+
+// PokedexFromFile return a list of Pokemon
+func PokedexFromFile() (pokes Pokedex, err error) {
+	rows, err := helpers.ReadFile(fmt.Sprintf("%s/pokemon.csv", constants.AssetsPath))
+
+	if err != nil {
+		return
+	}
+
+	pokes = make(Pokedex)
+
+	for _, row := range rows[1:167] {
```

Comment:
I understand why starting from 1 (to eliminate the columns' names), but why limit it to 167?

Reply:
In the csv file that I'm using, there are definitions for megaevolutions, so, I'm just skiping in next lines. 167 it's the row for mewtwo.